src/ACEtk/interpretation/ContinuousEnergyNeutron/Builder/Tabulated/src/ACEify.hpp

```diff
@@ -0,0 +1,19 @@
+void ACEify( Table::Data& tData ){
+  decltype( auto ) xss = tData.XSS();
+
+  if( bool( this->parameters ) ){
+    decltype( auto ) p = parameters.value();
+    xss.push_back( p.first.size() );
+    xss |= ranges::action::push_back( p.first );
+    xss |= ranges::action::push_back( p.second );
+  }
+  else{
+    xss.push_back( 0 );
+  }
+
+  xss.push_back( x.size() );
+  xss |= ranges::action::push_back( x );
+
+  details::ACEify( tData, this->y );
```

Comment:
Replace this with a aceify_y, to be implemented in the derived class to handle specifics of the vector<T> stuff (e.g. print NET for LAW=1)

Reply:
Had a "better" solution.